Model/UserManager.php

```diff
@@ -92,6 +92,7 @@ public function updateCanonicalFields(UserInterface $user)
     public function updatePassword(UserInterface $user)
     {
         $this->passwordUpdater->hashPassword($user);
+        $user->eraseCredentials();
```

Comment:
this should not be called here. The UserManager is not responsible for erasing credentials